components/kickofflabs/package.json

```diff
@@ -1,13 +1,12 @@
 {
```

Comment:
I think `@pipedream/platform` should be installed as a dependency for the package